janus.c

```diff
@@ -1310,6 +1311,21 @@ int janus_process_incoming_request(janus_request *request) {
 			type = NULL;
 			json_t *jsep_trickle = json_object_get(jsep, "trickle");
 			gboolean do_trickle = jsep_trickle ? json_is_true(jsep_trickle) : TRUE;
+			json_t *jsep_rids = json_object_get(jsep, "rid_order");
+			gboolean rids_hml = TRUE;
+			if(jsep_rids != NULL) {
+				const char *jsep_rids_value = json_string_value(jsep_rids);
+				if(jsep_rids_value != NULL) {
+					if(!strcasecmp(jsep_rids_value, "hml")) {
```

Comment:
nit: since you've already initialized it to TRUE, this branch has no effect and could be dropped.